### PR TITLE
Add a drone file to the project to trigger drone builds

### PR DESCRIPTION
**Pull request by JoeChapman**
Before drone will build the secrets need to be set and the project authorised to run on drone

- [x] Secrets have been added
- [x] Project has been authorised to run on drone

### CODE REVIEW

**Comment by lennym** [comment]
Maybe get rid of all the commented out stuff?

**Comment by JoeChapman** [comment]
@lennym fair point, but I have left it there for as soon as acceptance tests are added with a `Dockerfile-acceptance` file they'll be required again. So they are in situ for convenience.

**Comment by JoeChapman** [comment]
> Any compelling reason not to run acceptance tests?

There aren't any

**Comment by JoeChapman** [comment]
@lennym thanks for reviewing

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  # test_acceptance:
+  #   image: docker:1.11
+  #   environment:
+  #     - DOCKER_HOST=tcp://127.0.0.1:2375
+  #   commands:
+  #     - docker run -e NODE_ENV=ci -d --net host app
+  #     - docker run --rm --net host acceptance
+  #   when:
+  #     branch: master
+
```
Comment (sulthan-ahmed) [inline .drone.yml]: Was it intentional to have these comments here?

**Comment by lennym** [comment]
This *looks* fine to me, but also

**Pull request merged**